controllers/broadcast.go

```diff
@@ -71,7 +72,8 @@ func (broadcastController *BroadcastController) Post(w http.ResponseWriter, r *h
 	if err = broadcastController.MessageRepository.Create(message); err == nil {
 		logger.Info().Str(messageIDLogFieldKey, message.ID.String()).Msg("Message accepted for broadcast")
 		go broadcastController.Dispatcher.Dispatch(message)
-		writeStatus(w, http.StatusAccepted, nil)
+		w.Header().Add(headerLocation, "/channel/"+channel.ChannelID+"/message/"+message.MessageID)
```

Comment:
Location header should have absolute URL; to form the absolute URL, use the incoming URL to transform this relative URL to absolute.